### PR TITLE
[MEJB-129] Refactor - extract method

### PR DESCRIPTION
**Pull request by pzygielo**
- [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

**Comment by pzygielo** [comment]
May I ask for review, please?

File: src/main/java/org/apache/maven/plugins/ejb/IncludesExcludes.java [hunk 1]
```diff
@@ -46,33 +46,27 @@ public IncludesExcludes( List<String> includes, List<String> excludes, List<Stri
...
-        return result;
+        return resultingXcludes( includes, defaultIncludes );
     }
 
     public String[] resultingExcludes()
+    {
+        return resultingXcludes( excludes, defaultExcludes );
+    }
+
+    private static String[] resultingXcludes( List<String> currentXcludes, List<String> defaultXcludes )
```
Comment (michael-o) [inline src/main/java/org/apache/maven/plugins/ejb/IncludesExcludes.java:57]: can the args have a decent names?

Comment (pzygielo) [reply src/main/java/org/apache/maven/plugins/ejb/IncludesExcludes.java:57]: > can the args have a decent names?

Could you suggest something else that will cover both includes and excludes?

Comment (michael-o) [reply src/main/java/org/apache/maven/plugins/ejb/IncludesExcludes.java:57]: Forget my comment, it was a shortsight. I will rereview.

**Comment by michael-o** [comment]
Looking into...

**Comment by jira-importer** [comment]
Resolve #163

**Pull request closed without merging**